ci_action/main.py

```diff
@@ -166,6 +166,14 @@ def main():
         indented_errors = [textwrap.indent(e, '    ') for e in errors]
         enumerated_errors = [f' {i+1}. ' + e[4:] for i, e in enumerate(indented_errors)]
         error_list = '\n'.join(enumerated_errors)
+        LOG.error(f"Tests could not launch due to these errors:\n{error_list}")
+        return 1
+
+    if non_blocking_errors:
```

Comment:
@eap Can you provide an example of `error_list` please? I am having trouble following the string parsing below. Thanks!

Reply:
done